helpers/main.go

```diff
@@ -125,43 +124,11 @@ func validateLabelSelector(selector labels.Selector) error {
 	return nil
 }
 
-// transformCloudSpecToHostsSpec from a cloud spec disruption, get all ip ranges of services provided and transform them into a list of hosts spec
-func transformCloudSpecToHostsSpec(cloudManager *cloudservice.CloudServicesProvidersManager, cloudSpec *chaosv1beta1.NetworkDisruptionCloudSpec) ([]chaosv1beta1.NetworkDisruptionHostSpec, error) {
-	hosts := []chaosv1beta1.NetworkDisruptionHostSpec{}
-	clouds := cloudSpec.TransformToCloudMap()
-
-	for cloudName, serviceList := range clouds {
-		serviceListNames := []string{}
-
-		for _, service := range serviceList {
-			serviceListNames = append(serviceListNames, service.ServiceName)
-		}
-
-		ipRangesPerService, err := cloudManager.GetServicesIPRanges(cloudtypes.CloudProviderName(cloudName), serviceListNames)
-		if err != nil {
-			return nil, err
-		}
-
-		for _, serviceSpec := range serviceList {
-			for _, ipRange := range ipRangesPerService[serviceSpec.ServiceName] {
-				hosts = append(hosts, chaosv1beta1.NetworkDisruptionHostSpec{
-					Host:      ipRange,
-					Protocol:  serviceSpec.Protocol,
-					Flow:      serviceSpec.Flow,
-					ConnState: serviceSpec.ConnState,
-				})
-			}
-		}
-	}
-
-	return hosts, nil
-}
-
-// isModifiedError tells us if this error is of the form:
+// IsModifiedError tells us if this error is of the form:
 // "Operation cannot be fulfilled on disruptions.chaos.datadoghq.com "chaos-network-drop": the object has been modified; please apply your changes to the latest version and try again"
 // Sadly this doesn't seem to be one of the errors checkable with a function from "k8s.io/apimachinery/pkg/api/errors"
 // So we parse the error message directly
-func isModifiedError(err error) bool {
+func IsModifiedError(err error) bool {
```

Comment:
I have found a way to move properly all the method, but I have no idea where I should move this function.
We have an `utils` package where I can move it, but I think it is not really clean... 
Does anyone has an idea? @luphaz @Devatoria

Reply:
According to the comment, the initial purpose of this function really seems related to Disruption apparently.

Maybe `api/v1beta1` and confirming we are only using it for it's initial purpose.

I would not transform it to a method, we already have bunch of function there. Maybe renaming it could also discourage a broad usage.

Reply:
The function is used by [the chaos pod service](https://github.com/DataDog/chaos-controller/pull/764/files#diff-23f25f46d9eb125d7394bebc6d0438be79d6c2042d57dba8867e46eb7647a47aR352) and inside the [big catch error of the disruption controller](https://github.com/DataDog/chaos-controller/pull/764/files#diff-c1d39a98823ae5bac7f016a4cf1ae0fa77de72ead1f676208a83f056125ed6a5R114).

I can move it inside the [`validation.go`](api/v1beta1/validations.go) file of the `api/v1beta1` package. WDYT?

Reply:
I can also rename it by `IsConflcitError`. WDYT?